cmake/std/PullRequestLinuxDriverTest.py

```diff
@@ -193,6 +193,22 @@ def setBuildEnviron(arguments):
                      'sems-cmake/3.10.3',
                      'atdm-env',
                      'atdm-ninja_fortran/1.7.2'],
+                'Trilinos_pullrequest_gcc_8.3.0':
```

Comment:
We should have tests added for this change in unit_tests/TestPullRequestLinuxDriverTest.py. Otherwise looks good

Reply:
Ahh. Right. Addition pushed.